specs/binding-kafka.spec/src/main/scripts/io/aklivity/zilla/specs/binding/kafka/schema/kafka.schema.patch.json

```diff
@@ -343,81 +350,35 @@
                         {
                             "title": "Transforms",
                             "type": "array",
-                            "oneOf":
+                            "items":
```

Comment:
Let's keep the original as this is also locking down the supported order to match the implementation.
Later when we add more general support for transforms, we'll loosen this a bit more.

Reply:
The yaml validator doesn't like the redundant `oneOf` properties, because it can't tell the single prop and multiple prop options apart. If either of the single prop options are used the validator doesn't know which of the two options it could be. The order of the options didn't seem to change the outcome. 
![image](https://github.com/user-attachments/assets/9368f003-9691-4e90-958a-d3b02a5a2857)

Using an "anyOf" works but is the same thing as having two optional properties. An I am guessing the same issue might occur where the wrong options is used to validate if there isn't enough info for the validator to distinguish a difference.

Reply:
The concern is that by making the change we now validate variations that are not supported by the implementation. Can you confirm if the ordering requirement is retained or not by these proposed changes?
Btw, I don't think we get an issue with validation in `zilla` itself.